### DISCUSSION TITLE
Angular Module Federation application

### DISCUSSION THREAD

**Discussion by gregblade** [taiga-family/taiga-ui#6106] [Q&A] [answered]
Hi,
   We are trying to build Angular (v17) application with support of micro frontends using module federation. We have imported all modules but we are either getting errors or components are rendering correctly (or at all) in remotes (micro frontend apps). Any chance someone has guidelines or example on how to integrate Taiga-UI in that module federation?

![noworking](https://github.com/taiga-family/taiga-ui/assets/152197502/1cd24a06-6033-4889-a22a-d24152dce8c0)
![working](https://github.com/taiga-family/taiga-ui/assets/152197502/8fa42142-1e53-4d99-9dd1-10ac4164a90d)

**Comment by waterplea** [comment, marked as answer]
You need to share `@taiga-ui/polymorpheus` library too. Update everything to the latest version while you're at it, your screenshot suggests you got 2 polymorpheus in your code, it was recently updated and latest Taiga UI version got synced.
